SDL/LST.h

```diff
@@ -1,6 +1,8 @@
 #include <filesystem>
 #include <cstdlib>
 #include <numeric>
+#include <mutex>
+std::once_flag mapsLoaded, modulesInited;
```

Comment:
in cmssw I see that these are often declared just next line preceding `std::call_once` as `static std::once_flag name;`. So, would be in LST.cc

@dan131riley are there any downsides to keep the declaration of a `std::once_flag` next to the call where it's used?

BTW, I do not see any include guards in LST.h.
Is it really included only once? Otherwise we may have an ODR violation.

Reply:
It is included only once but I agree that it's better to add header guards. I can do so in an extra commit. Waiting for feedback on the static nature of the `std:once_flag`'s and I will push another commit before merging (in both PRs).

Reply:
Comment dealt in [6b3e014](https://github.com/SegmentLinking/TrackLooper/pull/315/commits/6b3e014f3d2cdd16ce82d8f5a0a519e842e82226). Code compiled (standalone and CMSSW) and CMSSW gave sensible results in a physics validation. So, barring any further comments, this is good to go on my side.
Once this is merged, I will propagate the extra commit to the master PR as well.

Reply:
This revision is definitely better, since it limits the scope of the flags to the places they are used.  We don’t like declarations with scope more extensive than necessary.